distributed/bokeh/templates/worker-table.html

```diff
@@ -20,7 +20,7 @@
         <td> {{ len(ws.processing) }} </td>
         <td> {{ len(ws.has_what) }} </td>
         {% if 'bokeh' in ws.services %}
-        <td> <a href="http://{{ ws.host }}:{{ ws.services['bokeh'] }}">bokeh</a> </td>
+        <td> <a href="http://{{ ws.host }}:{{ scheduler.services['bokeh'].port }}/proxy/{{ ws.services['bokeh'] }}/{{ ws.host }}/status">bokeh</a> </td>
```

Comment:
@mrocklin This is the new url: `/proxy/{port}/{host}/{proxied_path}`.

Reply:
Oh wow, how did you manage that?

Reply:
> @mrocklin I realized that I could manually overwrite the request uri (something I always had trouble with using ngxin) and a bit of regex in the handler made things smooth for having nicer URLs


Ah. Great!

Reply:
This is cool! 

Is ws.host coming from a trusted source? If it is user controlled, it could be used maliciously to bypass CORS / Origin checking.

Reply:
This is the address that a Dask worker advertises as its contact address.  A user that has access to launch a Dask worker and connect it to this scheduler can manipulate this value, yes.

However, such a user already has the ability to connec to the scheduler and run arbitrary Python code there, or on any other worker-connected machine, so I suspect that at this point we're ok with this.

Reply:
With the introduction of `scheduler` here we need to include that all uses of this template also include the `scheduler=` keyword, 

```diff
class Worker(RequestHandler):
    def get(self, worker):
        worker = escape.url_unescape(worker)
        with log_errors():
            self.render(
                "worker.html",
                title="Worker: " + worker,
                Worker=worker,
+               scheduler=self.server,
                **toolz.merge(self.server.__dict__, ns, self.extra)
            )
```

Reply:
This is something that should have shown up on tests ideally.  If you felt the desire, it might be nice to add this to some test that checks the info pages (assuming that something like this already exists)

Reply:
In my [last commit](https://github.com/dask/distributed/pull/2715/commits/d2d76e41b17f61350c99eb37787cc0c7204ccee6), I fixed the worker and task handlers.  The test were more or less already written in [test_scheduler_bokeh_html.py](https://github.com/dask/distributed/pull/2715/commits/d2d76e41b17f61350c99eb37787cc0c7204ccee6#diff-b5a38c24316b8b77073324cf2e59e29cR23) . The tests previously passed because the workers did not setup bokeh dashboards.  Now they do and tests pass again with the suggested addition above